wiki/osu!wiki_Contribution_Guide/Common_Mistakes/en.md

```diff
@@ -0,0 +1,30 @@
+# Common Mistakes
+
+*Main page: [osu!wiki Contribution Guide](/wiki/owcg).*
+
+## All of my commits say "Updated en.md"
+
+Or with other locale files followed by `.md`. This can also be phrased as, "Someone told me to use better commit messages."
+
+This means you didn't use good commit names when you were able/suppose to or you had attempted to move/edit multiple files using the GitHub web interface. Remember, commit names are supposed to be meaningful so anyone can understand what you changed without having to look at the GitHub diff themselves but be short enough to fit the character limit of 72 characters.
```

Comment:
This means you didn't use meaningful commit messages when you move/edit multiple files using the GitHub web interface. Remember, commit messages are supposed to be meaningful so anyone can understand what you changed without looking at the GitHub diff themselves. However, the commit messages should be short enough to fit the character limit of 72 characters.

Reply:
we your suggestion removed the fact that they might have not used good commit messages when vomiting locally... well consider rephrasing